.github/workflows/format-and-commit.yml

```diff
@@ -0,0 +1,37 @@
+name: Format And Commit
+
+concurrency:
+  group: ${{ github.ref }}
+
+on:
+    # Pending getting through individual PRs.
+    # push:
+    #   branches:
+    #     - master
+    workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  dprint-fmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+    - run: npm install
+    - uses: actions/cache@v3
+      with:
+        key: ${{ runner.os }}-dprint-${{ hashFiles('package-lock.json', '.dprint.jsonc') }}
+        path: ~/.cache/dprint
+        restore-keys: |
+          ${{ runner.os }}-dprint-
+    - run: npx dprint fmt
+    - uses: stefanzweifel/git-auto-commit-action@v4.16.0
+      with:
+        commit_author: "TypeScript Bot <actions@github.com>"
+        commit_message: "chore: npx dprint fmt"
```

Comment:
If the bot is doing the merging it can always send a GitHub API request before doing the merge API request... yet again I am griping that GitHub doesn't expose a hook to do this upon clicking the green button

Reply:
The bot won't be doing the merging of these formatting PRs as they are pushed straight to `master`, so it's not gonna retitle these changes.

I'm not a fan of the `chore:` prefix or `npx` being mentioned; we don't use those styles for the current commits we make, and we probably won't be using `npx` forever either.

Reply:
I really _really_ miss Codeflow's delineation between "resolved" and "closed"...